contracts/ROSCA.sol

```diff
@@ -245,7 +251,7 @@ contract ROSCA {
    * user with winnings
    */
   function cleanUpPreviousRound() internal {
-    uint256 winnerIndex;
+    uint256 winnerIndex = membersAddresses.length + 1; // to avoid default (0) to be winner
```

Comment:
I think it's better to have default winner rather than potentially no winner at all. Default of 0 is fine.

FYI, it's better to put the default in findSemiRandomWinner because winnerIndex is only effective if winnderAdress == 0 (i.e winnerSelectedThroughBid is false)

Reply:
reverted